README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+## Maddi Herne Biography
+My name is Maddi Herne. I am a 4th year majoring in *Public Policy*. I am from **Greenwich** in Connecticut, but I am French and German.  
+
+I do a lot on campus.
+* MUN
+  + ChoMUN
+  + Team
+  
+* Greek life
+
+* Pre-professional organizations
+  + UIBG
+  
+I am learning to cook using New York Times [recipes](https://cooking.nytimes.com).
+
+Here is my profile picture!
+
+![profile](https://github.com/maddiherne/hw01-1/blob/master/Screen%20Shot%202019-10-03%20at%203.00.17%20PM.png)
```

Comment:
- Great work! Especially, it is concise and organized using a lot of lists.
- It may be good if you use some headers to be more organized (eg. school life/ hobby ). 

----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     *      |              |            |
| **Coding strategy**         |    *       |              |            |
| **Presentation**            |        *   |              |            |
| **Achievement, creativity** |   *        |              |            |
| **Ease of access**          |          * |              |            |